packages/web-forms/src/components/controls/NoteControl.vue

```diff
@@ -1,24 +1,22 @@
 <script setup lang="ts">
 import type { NoteNode } from '@getodk/xforms-engine';
-import ControlLabel from '../ControlLabel.vue';
-import ControlHint from '../ControlHint.vue';
+import ControlText from '../ControlText.vue';
 
 defineProps<{question: NoteNode}>();
 </script>
 
 <template>
 	<div class="note-control">
-		<ControlLabel :question="question" />
-		<ControlHint :question="question" />
-	
+		<ControlText :question="question" />
+
 		<div v-if="question.currentState.value" class="note-value">
 			{{ question.currentState.value }}
 		</div>
 	</div>
 </template>
 
 <style scoped lang="scss">
-.note-control {
-	margin-bottom: 0.5rem;
-}
-</style>
\ No newline at end of file
+	:deep(.control-text) {
+		margin-bottom: 0;
+	}
```

Comment:
Really great example of why it makes sense for the container to control spacing rather than the contained thing.